### PR TITLE
Adkernel Bid Adapter: rtbdemand.com alias

### PR DESCRIPTION
**Pull request by ckbo3hrk**
## Type of change
- [x] Other

## Description of change
Adaptor alias for rtbdemand.com network

### CODE REVIEW

File: modules/adkernelBidAdapter.js [hunk 1]
```diff
@@ -97,7 +97,8 @@ export const spec = {
     {code: 'felixads'},
     {code: 'motionspots'},
     {code: 'sonic_twist'},
-    {code: 'displayioads'}
+    {code: 'displayioads'},
+    {code: 'rtbdemand_com'}
```
Comment (patmmccann) [inline modules/adkernelBidAdapter.js:101]: andbeyond team says they are not an alias of you; please remove if in error

please see @andbeyondmedia 

#9005

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/adkernelBidAdapter.js (hunk 1 above)
Comment (ckbo3hrk) [reply modules/adkernelBidAdapter.js:101]: Ok, we'll contact andbeyond team for confirmation

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/4063

File: modules/adkernelBidAdapter.js (hunk 1 above)
Comment (ckbo3hrk) [reply modules/adkernelBidAdapter.js:100]: @patmmccann andbeyond alias has been removed

**Review by patmmccann — APPROVED** [review]

**Pull request merged**